src/main/java/org/tron/common/net/udp/handler/MessageHandler.java

```diff
@@ -82,5 +82,6 @@ public void channelReadComplete(ChannelHandlerContext ctx) {
   public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
     logger.info("exception caught, {} {}", ctx.channel().remoteAddress(), cause.getMessage());
     ctx.close();
+    cause.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1224&issues=AWT5bJAqvlEfaeH0euXV&open=AWT5bJAqvlEfaeH0euXV)

[ISSUE_KEY]: AWT5bJAqvlEfaeH0euXV